test/e2e/accounts/common.ts

```diff
@@ -53,8 +53,13 @@ export async function installSnapSimpleKeyring(
 
   // navigate to test Snaps page and connect
   await driver.openNewPage(TEST_SNAPS_SIMPLE_KEYRING_WEBSITE_URL);
+
   await driver.clickElement('#connectButton');
 
+  if (process.env.ENABLE_MV3) {
+    console.log('Delay before connect button');
```

Comment:
Seems to be debugging logs? Should we get rid of those? (we have not other logging for `accounts` e2e tests)